XrmToolBox/New/NewForm.Designer.cs

```diff
@@ -927,6 +929,15 @@ private void InitializeComponent()
             this.checkForUpdateToolStripMenuItem.Text = "Check for update";
             this.checkForUpdateToolStripMenuItem.Click += new System.EventHandler(this.checkForUpdateToolStripMenuItem_Click);
             // 
+            // darkMode

```

Comment:
@MscrmTools  - I see you were down this road already, I updated the darkmode button. Do you think this should be the one and only way to trigger darkmode for plugins?

![image](https://github.com/MscrmTools/XrmToolBox/assets/42982326/a96a7022-852f-4a89-9e99-353a870ec3fd)